### DISCUSSION TITLE
Welcome to github-csv-tools Discussions!

### DISCUSSION THREAD

**Discussion by gavinr** [gavinr/github-csv-tools#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paul-improving** [comment]
Very handy tool @gavinr ! Thanks for creating! I ran into an issue importing issues from [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install) to GH where githubCsvTools kept telling me I needed a 'title' column when one already existed. I found a workaround in my CSV by moving the title column to the 3rd column, moving the 'body' column to the 2nd column, and inserting a blank column for column 1. Not sure if anyone has had that issue so wanted to share in case it comes up.

**Comment by cartokorn** [comment]
Very useful tool - thank you @gavinr! Using it every week!

Is there any possiblity assigning an issue to a project? https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects

That would be very helpful and even improve my weekly workflow!

Thanks in advance
